build.sbt

```diff
@@ -265,6 +265,21 @@ lazy val `example-todolist-client` = project
   .settings(exampleTodolistCommonSettings)
   .disablePlugins(ScriptedPlugin)
 
+/////////////////////
+//// MARSHALLERS ////
+/////////////////////
+
+lazy val `marshallers-jodatime` = project
```

Comment:
Should this really be in its own module? Makes sense if our practice is do so when introducing new dependencies, otherwise I suggest integrating in an existing module.

Reply:
In any case if making a new module, we should add it to the list in `quickstart.md`.

Reply:
I think it's a good idea to put it in it's own module in order to avoid adding more transitive dependencies. And yes, good catch, we should add it to the quickstart

Reply:
Sounds good. We could also add a sentence or two in the docs about the new marshallers.

Reply:
I'd add the information in the docs once we solved #269 because now it's not ready for its usage.  @L-Lavigne @juanpedromoreno thoughts?